client/app/webUniverseComponents/guides/guides.js

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
extra empty line.

Reply:
should be named as `client/app/webUniverseComponents/guides/guides.directive.js`